src/ansys/fluent/visualization/__init__.py

```diff
@@ -61,5 +62,152 @@ def version_info() -> str:
 )
 from ansys.fluent.visualization.graphics import Graphics  # noqa: F401
 from ansys.fluent.visualization.plotter import Plots  # noqa: F401
-from ansys.fluent.visualization.registrar import get_renderer, register_renderer
+from ansys.fluent.visualization.registrar import (
+    _visualizer,
+    get_renderer,
+    register_renderer,
+)
 from ansys.fluent.visualization.renderer import GraphicsWindow
+
+
+class View(str, Enum):
+    """Available view options."""
+
+    XY = "xy"
+    XZ = "xz"
+    YX = "yx"
+    YZ = "yz"
+    ZX = "zx"
+    ZY = "zy"
+    ISOMETRIC = "isometric"
+
+
+class _ViewWrapper:
+    def __init__(self, default=View.ISOMETRIC):
+        self._value = default
+
+    def __call__(self):
+        return self._value
+
+    def __str__(self):
+        return str(self._value)
+
+    def __repr__(self):
+        return repr(self._value)
+
+    def __eq__(self, other):
+        return self._value == other
+
+    def __getattr__(self, attr):
+        # Allows access like config.view.name or config.view.value
+        return getattr(self._value, attr)
+
+    def allowed_values(self):
+        """List of allowed views."""
+        return list(View)
+
+    def set(self, val):
```

Comment:
@prmukherj Using `set` as a method name can be confusing and even potentially hazardous (e.g., aliasing `set = obj.set` shadows the built-in).

Reply:
Removed _ViewWrapper